src/Stratis.Bitcoin.Features.PoA/Voting/VotingManager.cs

```diff
@@ -429,6 +429,14 @@ private void OnBlockConnected(BlockConnected blockConnected)
                         {
                             ChainedHeader chainedHeader = chBlock.ChainedHeader.GetAncestor(poll.PollStartBlockData.Height);
 
+                            if (chainedHeader?.Header == null)
```

Comment:
If this is null, this would still stop the sync wont it? I'm wondering whether or not we should perhaps check if the poll was done on a fork.... could this be the cause for the null ancestor?

Reply:
I doubt that a fork would lead to `null` as a block should exist at the height regardless. 

The changes in the code are intended to be released together with any other changes to help us identify the root cause.